Sources/OpenAPI/.create-api.yml

```diff
@@ -1,46 +1,21 @@
-# Modifier for all generated declarations
-access: public
-# Add @available(*, deprecated) attributed
-isAddingDeprecations: true
-# Available values: ["spaces", "tabs"]
+generate: [entities, paths, enums]
+module: AppStoreConnect_Swift_SDK
 indentation: tabs
-# Parses dates (e.g. "2021-09-29") using `NaiveDate` (https://github.com/CreateAPI/NaiveDate)
-isNaiveDateEnabled: false
-# Overrides file header comment
-fileHeaderComment: "// Generated by Create API\n// https://github.com/CreateAPI/CreateAPI"
-
+useNaiveDate: false
+fileHeaderComment: |
+  // Generated by Create API
+  // https://github.com/CreateAPI/CreateAPI
+  //
+  // swiftlint:disable all
```

Comment:
The old `isSwiftLintEnabled` option was removed, so to continue suppressing the linter warnings the old way, I've added this to the header. 

Alternatively you could use a **.swiftlint.yml** file to just exclude this Generated directory.